diff --git a/src/condor_examples/condor_config.generic b/src/condor_examples/condor_config.generic
index a1c17e6..aa823c4 100644
--- a/src/condor_examples/condor_config.generic
+++ b/src/condor_examples/condor_config.generic
@@ -48,3 +48,6 @@ use SECURITY : HOST_BASED
 ##--------------------------------------------------------------------
 ## Values set by the condor_configure script:
 ##--------------------------------------------------------------------
+
+# OSG customization - use blahpd from system.
+BATCH_GAHP = $(BIN)/blahpd
